### PR TITLE
Feat/st changes: SNI-4321 Merge changes to PET Release branch

### PR DESCRIPTION
**Pull request by sjamihmcts**
### Change description ###

Changes to merge ST related changes to PET release branch

### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/SNI-4321

**Before merging a pull request make sure that:**

- [x ] tests have been updated / new tests has been added (if needed)
- [x ] README and other documentation has been updated / added (if needed)

**If this ticket will have any visible impact on users and is not behind a feature toggle, make sure that:**
- [ ] this ticket been reviewed by QA
- [ ] the user story been signed off by the PO

Note that bug fixes, dependency updates and technical tasks do not directly impact the user experience and can be merged without QA and PO review.

### CODE REVIEW

**Review by muradalamv1 — APPROVED** [review]

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerCreateDraftOrder.java [hunk 1]
```diff
@@ -164,8 +165,9 @@ public AboutToStartOrSubmitResponse<CaseData, State> midEvent(
         CaseDetails<CaseData, State> detailsBefore
     ) {
 
-        var caseData = orderService.generateOrderFile(details.getData(), details.getId());
-
+        Calendar cal = Calendar.getInstance();
+        String date = simpleDateFormat.format(cal.getTime());
+        var caseData = orderService.generateOrderFile(details.getData(), details.getId(), date);
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerCreateDraftOrder.java:170]: Can we not use Calendar? It's really well out of date. Use LocalDate.

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerEditDraftOrder.java [hunk 2]
```diff
@@ -90,7 +95,9 @@ public AboutToStartOrSubmitResponse<CaseData, State> midEvent(
         CaseDetails<CaseData, State> detailsBefore
     ) {
 
-        var caseData = orderService.generateOrderFile(details.getData(), details.getId());
+        Calendar cal = Calendar.getInstance();
+        String date = simpleDateFormat.format(cal.getTime());
```
Comment (osheaf01) [inline src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerEditDraftOrder.java:99]: As above, can we not use Calendar?

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerCreateDraftOrder.java (hunk 1 above)
Comment (sjamihmcts) [reply src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerCreateDraftOrder.java:170]: Changed from Calendar to LocalDateTime.

File: src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerEditDraftOrder.java (hunk 2 above)
Comment (sjamihmcts) [reply src/main/java/uk/gov/hmcts/sptribs/caseworker/event/CaseWorkerEditDraftOrder.java]: Changed from Calendar to LocalDateTime.

**Review by osheaf01 — DISMISSED** [review]
LGTM

**Review by muradalamv1 — APPROVED** [review]

**Review by osheaf01 — APPROVED** [review]
LGTM

**Pull request merged**